crates/nu-command/tests/commands/hash_/mod.rs

```diff
@@ -43,19 +43,17 @@ fn error_when_invalid_character_set_given() {
         .contains("this is invalid is not a valid character-set"));
 }
 
-// FIXME: jt: needs more work
-#[ignore]
 #[test]
 fn base64_decode_characterset_binhex() {
     let actual = nu!(
         cwd: ".", pipeline(
         r#"
-        echo "F@0NEPjJD97kE'&bEhFZEP3" | hash base64 --character_set binhex --decode
+        echo "F@0NELS[A@0bFepaB!" | hash base64 --character-set binhex --decode
```

Comment:
i'd prefer to leave the original string because we have to be able to accomodate a `'` inside of a string like we did before. this change just hides that bug.

Reply:
Totally agree with you